addon/components/fast-link-to.js

```diff
@@ -1,75 +1,83 @@
 import Ember from 'ember';
-import layout from '../templates/components/fast-link-to';
-
-let { Component, generateGuid, get, getOwner, LinkComponent } = Ember;
-
-function findOriginal(PrototypeMixin) {
-  if (!PrototypeMixin.ownerConstructor) { return; }
-
-  let result;
-  let identity = PrototypeMixin.ownerConstructor.toString();
-  if (identity === 'LinkComponent') {
-    return PrototypeMixin.mixins[1].properties;
-  } else {
-    for (let i = 0; i < PrototypeMixin.mixins.length; i++) {
-      result = findOriginal(PrototypeMixin.mixins[i]);
-      if (result) { return result; }
-    }
+
+export default Ember.LinkComponent.extend({
+  init() {
+    this._super(arguments);
```

Comment:
this line is incorrect, it should be: https://github.com/nathanhammond/ember-fast-link-to/pull/12/files#diff-a2ce99631ee4265e14624fa994eb85c7R5

`this._super(...arguments)`